tests/rados/rados_prep.py

```diff
@@ -0,0 +1,327 @@
+        log.error(f"Did not find the log files : {files}")
+        return False
+    return True
+
+
+def enable_email_alerts(node: CephAdmin, **kwargs) -> bool:
+    """
+    Enables the email alerts module and configures alerts to be sent
+    Args:
+        node: Cephadm node where the commands need to be executed
+        **kwargs: Any other param that needs to be set
+
+    Returns: True -> pass, False -> fail
+    """
+    try:
+        cmd = "ceph mgr module enable alerts"
+        node.shell([cmd])
+
+        cmd = "ceph config set mgr mgr/alerts/smtp_host smtp.corp.redhat.com"
+        if kwargs.get("smtp_host"):
+            cmd = f"ceph config set mgr mgr/alerts/smtp_host {kwargs.get('smtp_host')}"
+        node.shell([cmd])
+
+        cmd = "ceph config set mgr mgr/alerts/smtp_sender ceph-iad2-c01-lab.mgr@redhat.com"
+        if kwargs.get("smtp_sender"):
+            cmd = f"ceph config set mgr mgr/alerts/smtp_sender {kwargs.get('smtp_sender')}"
+        node.shell([cmd])
+
+        cmd = "ceph config set mgr mgr/alerts/smtp_ssl false"
+        if kwargs.get("smtp_ssl"):
+            cmd = "ceph config set mgr mgr/alerts/smtp_ssl true"
+        node.shell([cmd])
+
+        cmd = "ceph config set mgr mgr/alerts/smtp_port 25"
+        if kwargs.get("smtp_port"):
+            cmd = f"ceph config set mgr mgr/alerts/smtp_port {kwargs.get('smtp_port')}"
+        node.shell([cmd])
+
+        cmd = "ceph config set mgr mgr/alerts/interval 5"
+        if kwargs.get("interval"):
+            cmd = f"ceph config set mgr mgr/alerts/interval {kwargs.get('interval')}"
+        node.shell([cmd])
+
+        cmd = "ceph config set mgr mgr/alerts/smtp_from_name 'Rados 5.0 sanity Cluster'"
+        if kwargs.get("smtp_from_name"):
+            cmd = f"ceph config set mgr mgr/alerts/smtp_from_name '{kwargs.get('smtp_from_name')}'"
+        node.shell([cmd])
+
+        if kwargs.get("smtp_destination"):
+            for email in kwargs.get("smtp_destination"):
+                cmd = f"ceph config set mgr mgr/alerts/smtp_destination {email}"
+                node.shell([cmd])
+        else:
+            log.error("email addresses not provided")
+            return False
+
+    except Exception as err:
+        log.error("Error while configuring the cluster for email alerts")
+        log.error(err)
+        return False
+
+    # Printing all the configuration set for email alerts
+    cmd = "ceph config dump | grep 'mgr/alerts'"
+    log.info(node.shell([cmd]))
+
+    # Disabling and enabling the email alert module after setting all the config
+    try:
+        cmd = "ceph mgr module disable alerts"
+        node.shell([cmd])
+        time.sleep(2)
+        cmd = "ceph mgr module enable alerts"
+        node.shell([cmd])
+        time.sleep(2)
+    except Exception as err:
+        log.error("Error while enabling/disabling alerts module after configuration")
+        log.error(err)
+        return False
+
+    # Triggering email alert
+    try:
```

Comment:
I believe the below statements can be part of a single try/catch

Reply:
There where separate errors seen with enabling and disabling the module, and instructing the module to force send a email. To differentiate b/w the point of failure, placed them in separate try bocks.